app/core/constants.js

```diff
@@ -88,6 +88,6 @@ export const DIABETES_DATA_TYPES = [
   'physicalActivity',
 ];
 
-export const MGDL_UNITS = t('mg/dL');
-export const MMOLL_UNITS = t('mmol/L');
+export const MGDL_UNITS = 'mg/dL';
+export const MMOLL_UNITS = 'mmol/L';
```

Comment:
Does it mean that those units won't be translated ?

Reply:
Yes, but since theses constants are used to get high/low values, in tidelinedata for example,
we can't translate them without making tidelinedata crash.
If we want to translate, we have to do it in the views at fly.
Anyway, the way the were translated won't take the language change.
Shall I change the views?

Reply:
something that we have to keep in mind. 
I close it for now.